update/verify_upgrade_db.sh

```diff
@@ -154,4 +158,124 @@ do
    /usr/bin/psql --host $HOST --port 5432 --username "$USER" --no-password -d "$QWATSERVICETEST" -f $i
 done
 
-exit $EXITCODE
+
+cd ..
+
+
+if [[ $EXITCODE == 0 ]]; then
+    if [[ $TRAVIS_BRANCH == 'master' ]]; then
+        printf "\n\n\n${GREEN}Updating DATA-SAMPLE${NC}\n\n\n"
+
+        TAB_FILES_POST=()
+        # If all is OK, update the DUMP demo ONLY IF WE ARE in the master
+        # 1 - Load the DEMO dump in a new demo DB
+        printf "${YELLOW}Creating DB (qwat_demo)${NC}\n"
+        /usr/bin/createdb "$DEMODB" --host $HOST --port 5432 --username "$USER" --no-password
+        /usr/bin/psql --host $HOST --port 5432 --username "$USER" --no-password -d "$DEMODB" -c "CREATE EXTENSION postgis; CREATE EXTENSION hstore;"
+
+        printf "\n${YELLOW}Cloning Data-sample repository${NC}\n"
+
+        #git remote add upstream "https://$GH_TOKEN@github.com/rust-lang/rust-by-example.git"
+    #     git config user.name "$QWAT_USER"
+    #     git config user.email "$QWAT_EMAIL"
+    #     git config --global push.default simple
+
+        git clone https://$GH_TOKEN@github.com/qwat/qwat-data-sample.git data-sample
```

Comment:
might be worth adding `--depth 1` to avoid fetching too much data. not sure it's a big deal though.